main.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import tensorflow as tf
+import numpy as np
+import cv2
+
+def load_pb(path_to_pb):
+    with tf.io.gfile.GFile(path_to_pb, "rb") as f:
+        graph_def = tf.compat.v1.GraphDef()
+        graph_def.ParseFromString(f.read())
+    with tf.Graph().as_default() as graph:
+        tf.import_graph_def(graph_def, name='')
+        return graph
+
+def main(argv):
+    graph = load_pb(argv[0])
+
+    labels = []
+    labels_filename = argv[2]
+
+    # Create a list of labels.
+    with open(labels_filename, 'rt') as lf:
+        for l in lf:
+            labels.append(l.strip())
+   
+    imageFile = argv[1]
+    image = cv2.imread(imageFile)
+
+    print(np.shape([image]))
+
+    image_mean = 127.5
+    image_std = 127.5
+    image_size_x = 299
+    image_size_y = 299 
+
+    input_image = cv2.resize(image, (image_size_x, image_size_y), interpolation = cv2.INTER_LINEAR)
+    # print([input_image])
+    print(np.shape([input_image]))
+
+    # img = [[[[0 for i in range(1)] for x in range(image_size_x)] for y in range(image_size_y)]for z in range(3)]
+
+    img = np.zeros((1,299,299,3))
+    
+    for i in range(image_size_x):
+        for j in range(image_size_y):
+            pixelValue = input_image[i][j]
+            # print()
+            # print(pixelValue)
+            # print((pixelValue - image_mean)/ image_std)
+            img[0, i, j] = (pixelValue - image_mean) / image_std
```

Comment:
Original resized image has 229x229x3 shape. To change layout, use
```python
img = img.transpose(2, 0, 1)
```
then
```python
img = img.reshape(1, 3, 229, 229)
```
Arithmetic operations also can be done by `img = (img - image_mean) / image_std)`

Reply:
![image](https://user-images.githubusercontent.com/32199725/80013373-6ffe6700-84d7-11ea-96ca-45ff5f5bc2f1.png)
Are you sure we have to do transponse?
https://github.com/tensorflow/models/blob/master/research/slim/nets/resnet_v2.py

Reply:
That's right. Sure, no transpose is necessary.